File: tests/go/cases/case_071_snapshot_profile.go
package escape_tests

var retainedCase071 = []map[string]string{}

func Case071SnapshotProfile(input string) string {
    // Task case 071: snapshot profile payload for downstream business processing.
    raw := input
    if raw == "" {
        raw = "sample"
    }
    payload := map[string]string{
        "task": "snapshot_profile",
        "entity": "profile",
        "stage": "snapshot",
        "input": raw,
    }
    // ESCAPE: payload is promoted to package-level retained cache.
    retainedCase071 = append(retainedCase071, payload)
    return "ok"
}

File: tests/java/src/main/java/com/escape/tests/cases/Case071SnapshotProfile.java
package com.escape.tests.cases;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.function.Supplier;

/** Task case 071: snapshot profile payload for downstream business processing. */
public class Case071SnapshotProfile {
    private static final Map<String, Map<String, String>> RETAINED_CACHE = new HashMap<>();
    private static final List<Map<String, String>> RETAINED_AUDIT = new ArrayList<>();
    private static final List<Supplier<String>> RETAINED_HANDLERS = new ArrayList<>();

    public static String execute(String input) {
        // Task: snapshot profile records and prepare transport-ready payload.
        String taskName = "snapshot_profile";
        String raw = (input == null || input.isEmpty()) ? "sample" : input;
        Map<String, String> payload = new HashMap<>();
        payload.put("task", taskName);
        payload.put("entity", "profile");
        payload.put("stage", "snapshot");
        payload.put("input", raw);
        // ESCAPE: payload is promoted to class-level retained cache.
        RETAINED_CACHE.put("case_071", payload);
        return "ok";
    }
}
